### ISSUE TITLE
Feedback Task 14

### ISSUE THREAD

**Issue by aufaroot18** [bangfarhan/Frontend-STTNF-SMT4#6]
- Kenapa project aku yang dideploy?

**Comment by farhanlubold** [comment]
Maaf ka aku salah paham, dikira uploadan yg di github bang aufa itu emg
disediain buat default program. Jadi farhan kalau mau ngerjain clone punya
kak aufa biar lebih rapih 🙏

Pada tanggal Jum, 24 Jun 2022 15.04, Aufa Billah ***@***.***>
menulis:

**Comment by aufaroot18** [comment]
No, make project sendiri. Source code aku cuma untuk referensi belejar bukan digunakan untuk diupload sbg tugas. :)
